main.c

```diff
@@ -60,11 +59,11 @@ int main(int argc, char** argv) {
         void xyzOrthographicProjection();
 
         //~~~~~ Group 17 ~~~~~//
-        int outputPoint(struct point * finalPoint, char* filename);
+        outputPoints(fileName);
```

Comment:
can you free the objects malloced in here (i know they arent malloced yet because of group 1 but for now could you just add a block of code under group 1 instantiating the points malloced? this will also resolve the other issues you were having